pyanaconda/errors.py

```diff
@@ -222,28 +222,64 @@ def _noSuchPackageHandler(self, exn):
+        broken_groups_modules = exn.error_group_specs
+
+
         message = ""
-        if exn.missing_packages:
-            if len(exn.missing_packages) > 1:
-                packages = ", ".join(exn.missing_packages)
+
+        # missing packages/groups/modules
+
+        if missing_packages:
+            if len(missing_packages) > 1:
+                packages = ", ".join(missing_packages)
                 message = message + _("The following packages are missing:\n%s\n\n") % packages
             else:
-                message = message + _("The following package is missing:\n%s\n\n") % exn.missing_packages[0]
+                message = message + _("The following package is missing:\n%s\n\n") % missing_packages[0]
 
-        if exn.missing_groups_and_modules:
-            if len(exn.missing_groups_and_modules) > 1:
-                groups_modules = ", ".join(exn.missing_groups_and_modules)
+        if missing_groups_modules:
+            if len(missing_groups_modules) > 1:
+                groups_modules = ", ".join(missing_groups_modules)
                 message = message + _("The following groups or modules are missing:\n%s\n\n") % groups_modules
             else:
-                message = message + _("The following group or module is missing:\n%s\n\n") % exn.missing_groups_and_modules[0]
+                message = message + _("The following group or module is missing:\n%s\n\n") % missing_groups_modules[0]
 
-        message = message + _("Would you like to ignore this and continue with installation?")
+        # broken packages/groups/modules
 
-        if self.ui.showYesNoQuestion(message):
-            return ERROR_CONTINUE
-        else:
+        if broken_packages:
+            if len(broken_packages) > 1:
+                packages = ", ".join(broken_packages)
+                message = message + _("The following packages are broken:\n%s\n\n") % packages
+            else:
+                message = message + _("The following package is broken:\n%s\n\n") % broken_packages[0]
+
+        if broken_groups_modules:
+            if len(broken_groups_modules) > 1:
+                groups_modules = ", ".join(broken_groups_modules)
+                message = message + _("The following groups or modules are broken:\n%s\n\n") % groups_modules
+            else:
+                message = message + _("The following group or module is broken:\n%s\n\n") % broken_groups_modules[0]
+
+        # if we have at least one broken package, group or module we will abort the installation
+        if broken_packages or broken_groups_modules:
+            broken_item_count = len(broken_packages) + len(broken_groups_modules)
+            if broken_item_count >1:
+                message = message + _("Some of packages, groups or modules are broken, the installation will be aborted.")
+            else:
+                message = message + _("Some of packages, groups or modules are broken, the installation will be aborted.")
```

Comment:
Please, remove the messages for just one package/group/module. I think that a message like `The following packages are broken: python` is readable enough.

Btw. the message on this line is even the same as the previous one.

Reply:
OK, it will be simpler & there will be less strings to translate. :)

Reply:
It looks better, but you can simplify also the part for `Some of packages, groups or modules are broken, the installation will be aborted`.

Reply:
Oh, right, done.